app/scripts/components/dashboard/charts/waterpoint-functional-chart.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropTypes } from 'react';
+import {BarChart} from 'react-d3';
+import * as func from '../../../utils/functional';
+
+require('stylesheets/dashboard/charts/waterpoint-functional-chart');
+
+const WaterpointFunctionalChart = React.createClass({
+  propTypes: {
+    waterpoints: PropTypes.array.isRequired,
+  },
+
+  parseData(waterpoints) {
+    const comparator = (a, b) => b.y - a.y;
+    return [{
+      name: '% Functional',
+      values: Object.keys(waterpoints).map(key => {
+        return {
+          x: key,
+          y: (waterpoints[key].FUNCTIONAL / waterpoints[key].total * 100),
+        };
+      }).sort(comparator),
+    }];
+  },
+
+  render() {
+    const waterpointsRes = func.Result.countByGroupBy(this.props.waterpoints, 'REGION', 'STATUS');
+    return (
+      <div className="waterpoint-functional-chart">
+        <BarChart
```

Comment:
Wrap `<BarChart />` with `<TSetChildProps>` to translate it, like `<TSetChildProps><BarChart title={{k='chart.functional-waterpoints'}} ... /></TSetChildProps>`